tests/conftest.py

```diff
@@ -106,7 +106,9 @@ class DictBot(Bot):
 
 @pytest.fixture(scope='session')
 def bot(bot_info):
```

Comment:
Maybe use the builder here as well?

Reply:
`Bot` and `ExtBot` currently don't have builders. See my first bullet point in the PR description.

if we want to add a builder for `tg.Bot`, this can only be optional, because one needs to be able to instantiate `tg.Bot` without `tg.ext`